esp/src/src-react/components/IndexFileSummary.tsx

```diff
@@ -103,9 +113,7 @@ export const IndexFileSummary: React.FunctionComponent<IndexFileSummaryProps> =
             onClick: () => {
                 file?.update({
                     UpdateDescription: true,
-                    FileDesc: description,
-                    Protect: _protected ? WsDfu.DFUChangeProtection.Protect : WsDfu.DFUChangeProtection.Unprotect,
-                    Restrict: restricted ? WsDfu.DFUChangeRestriction.Restrict : WsDfu.DFUChangeRestriction.Unrestricted,
```

Comment:
This was the change that I mentioned in my commit message which was from HPCC-34159 which removed these lines from LogicalFileSummary.tsx